### DISCUSSION TITLE
How do I switch the Bito.ai plugin in IntelliJ to ligth theme?

### DISCUSSION THREAD

**Discussion by hrstoyanov** [gitbito/bitoai#83] [General]
... for some reason, it is always dark themed?

**Comment by monicfenga** [comment]
They have been included in their planning, see https://github.com/gitbito/bitoai/issues/16 for details

**Comment by nisha-bito** [comment]
hi @hrstoyanov / @monicfenga ,

Bito Plugin now supports light and Dark theme both. Please refer to documentation - https://docs.bito.ai/feature-guides/ai-chat-in-bito/appearance-settings. 
and do let us know if you have any further questions on this. 

Thanks,
Nisha
